### PR TITLE
Add party.supertux.supertuxparty

### PR DESCRIPTION
**Pull request by Dragoncraft89**
Super Tux Party is a free/libre and open-source party game that is meant to replicate the feel of games such as Mario Party

This version still uses the godot baseapp, as the submodule version isn't available yet. Will change that when it becomes available

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build party.supertux.supertuxparty

**Comment by nedrichards** [comment]
bot, build party.supertux.supertuxparty

**Comment by LeandroStanger** [comment]
bot, build party.supertux.supertuxparty

**Comment by LeandroStanger** [comment]
bot, build party.supertux.supertuxparty

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"skip_arches": ["aarch64"]
+}
```
Comment (LeandroStanger) [inline flathub.json:3]:
```suggestion
{
  "skip-arches": ["aarch64"]
}
```

**Comment by barthalion** [comment]
bot, build party.supertux.supertuxparty

File: party.supertux.supertuxparty.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  <url type="translate">
+	https://hosted.weblate.org/projects/super-tux-party
+  </url>
+
+  <screenshots>
+	<screenshot type="default">
+	  <image> http://supertux.party/screenshots/1.png </image>
+	</screenshot>
+	<screenshot type="default">
+	  <image> http://supertux.party/screenshots/2.png </image>
```
Comment (AsciiWolf) [inline party.supertux.supertuxparty.metainfo.xml:46]: This screenshot is too small.

Comment (Dragoncraft89) [reply party.supertux.supertuxparty.metainfo.xml:46]: Screenshot is updated with a higher res version

**Pull request closed without merging**